package.json

```diff
@@ -2,17 +2,22 @@
   "name": "wdio-cucumberjs-json-reporter",
   "version": "2.0.4",
```

Comment:
Hi @yon-cuadrado 

Thanks for the PR and all the work you've put into it. I just want to check-check-double-check before we merge this. The current version is version 3.0.0, and the package.json is at least at `2.0.4`, which one was your main branch? Are you sure all latest changed are in this one?
If so, can you upgrade it to 3.0.0 so the package.json file is also in line with the main branch?

Thanks

Reply:
I used the branch main as the starting point for the pr. I have left the version value in the package.json as it is in this branch, what value should I put?, 3.0.1 ?

Reply:
Hello
Thanks for taking the time to review the pr.
The branch from where I started my pr is main. I have updated the package.json version to 3.0.0. I think that I have all the changes included but as this is my first pr in the project I think it would be great if you could take a closer look just in case I have missed any other thing.

Regards